adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -21,6 +32,98 @@ table below.
 
 == Usage example
 
+The example below rewrites the application from
+<<subsec:example.sycl.application>> to demonstrate the usage of this extension.
+
+[source,role=synopsis]
+----
+#include <iostream>
+#include <sycl/sycl.hpp>
+using namespace sycl;  // (optional) avoids need for "sycl::" before SYCL names
+
+int main() {
+  // Create a queue to work on.
+  queue myQueue;
+  
+  khr::command_graph myGraph{myQueue};
+
+  // Create some 2D buffers of float for our matrices
+  buffer<float, 2> a{range<2>{N, M},
+                     khr::property::graph::assume_buffer_outlives_graph{}};
+  buffer<float, 2> b{range<2>{N, M},
+                     khr::property::graph::assume_buffer_outlives_graph{}};
+  buffer<float, 2> c{range<2>{N, M},
+                     khr::property::graph::assume_buffer_outlives_graph{}};
+
+  myGraph.begin_recording(myQueue);
```

Comment:
we should put the graph usage code inside a scope to grantee it will we destroyed before the buffers, e.g https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Graph/Inputs/basic_buffer.cpp#L27